cmd/antrea-agent/agent.go

```diff
@@ -114,7 +114,6 @@ func run(o *Options) error {
 	if err != nil {
 		return fmt.Errorf("error creating K8s clients: %v", err)
 	}
-	k8s.OverrideKubeAPIServer(o.config.KubeAPIServerOverride)
```

Comment:
I suggested moving the validation of `o.config.KubeAPIServerOverride` to options.go, but overriding the env variable should still run here as it will be harder to find where the env variables are overridden otherwise. And it would be surprising that a "validate" function has a side effect of setting the env variables.

Reply:
Updated,  thanks.